modules/nf-core/bwa/index/main.nf

```diff
@@ -11,7 +11,7 @@ process BWA_INDEX {
     tuple val(meta), path(fasta)
 
     output:
-    tuple val(meta), path(bwa) , emit: index
+    tuple val(meta), path("./bwa") , emit: index
```

Comment:
`./bwa`?

Reply:
the language server thought the unquoted `bwa` was a variable